config/settings.yml

```diff
@@ -80,3 +80,6 @@ plugins:
     default: 10000
     hidden: true
     client: true
+  min_trust_to_flag_chat_messages:
```

Comment:
I think we should follow the pattern we have been doing in core recently, and no longer use trust-based settings. Instead use a `group_list` setting called e.g. `chat_message_flag_allowed_groups` and default it to auto group ID 11 (TL1). The PR I linked in core with another comment also makes a special shortcut method for group_list settings, adding `_map` to the end of the name to get an array of integers instead of having to do the `split("|")` thing all the time, .e.g `chat_message_flag_allowed_groups_map` and then you can use `user.in_any_groups?` too

Reply:
agreed group based is much better